### DISCUSSION TITLE
Av error

### DISCUSSION THREAD

**Discussion by MohammadReza0GH** [phoebe-project/phoebe2#859] [Questions]
when i change Av this error appears:

ERROR: 'Johnson:V' passband (passband@lc02@lc@dataset) does not support ck2004:ext (atm@_default@phoebe01@phoebe@compute). (2 affected parameters, affecting run_compute)
ERROR: 'Johnson:V' passband (passband@lc02@lc@dataset) does not support blackbody:ext (atm@primary@phoebe01@phoebe@compute). (2 affected parameters, affecting run_compute)
ERROR: 'Johnson:V' passband (passband@lc02@lc@dataset) does not support blackbody:ext (atm@secondary@phoebe01@phoebe@compute). (2 affected parameters, affecting run_compute)

i changed passband and atm model but error remains.

**Comment by Raindogjones** [comment]
This looks to me like, for some reason, the passband file doesn't contain the extinction tables and isn't downloading them on the fly. To diagnose why, can you please provide a minimal script that results in this error?

**Reply by MohammadReza0GH** [reply]
import phoebe
b = phoebe.default_binary()
logger = phoebe.logger(clevel = 'WARNING')

b.set_value('Av', value=0.8)
b.add_constraint('semidetached', 'secondary')
b.set_value('period@binary', value = 0.1725)
b.set_value('incl@binary@orbit', value = 78.1)
b.set_value('teff', component = 'secondary', value = 6383)
b.set_value('teff', component = 'primary', value = 100000)
b.set_value('q', value = 0.066)
b.set_value('requiv', component = 'primary', value = 0.00719)
b.set_value('sma', component = 'binary', value= 0.22)
b.set_value('eclipse_method', value ='only_horizon')
b.set_value('distance@system', value = 2060, unit = 'pc')

b.add_dataset('lc', compute_times=phoebe.linspace(0,0.1725,101),ld_mode='manual',pblum_mode='absolute',dataset= 'lc01')
b.run_compute(irrad_method='none')

**Reply by Raindogjones** [reply]
Ok, it does indeed seem like phoebe isn't recognising the missing tables and automatically downloading them, I suspect because of a problem with the passbands on the server being incomplete. In the mean time, you might be able to get around this by selecting another passband (for example, it seems as though 'SDSS:g' is complete).

Paging @kecnry and @aprsa: It looks to me as though extinction is missing in some of the online passbands? Some have atm:ext (where atm is bb, ck2004, phoenix) in their contents but others don't and Johnson:V is one where this is missing.

**Reply by MohammadReza0GH** [reply]
thank you, how can i fix it?

**Reply by Raindogjones** [reply]
We're working on migrating the passband server (and at the same time updating these passbands), so for now I'm afraid you will just have to use a different passband (e.g. "SDSS:g") or wait for the server migration.

**Reply by aprsa** [reply]
@MohammadReza0GH, is it just the Johnson V passband that you need extinction for?
